py3status/module.py

```diff
@@ -71,7 +71,9 @@ def __init__(self, module, user_modules, py3_wrapper, instance=None):
         self._py3_wrapper = py3_wrapper
 
         try:
+            error_msg = u"{}: Import Error, {}"
             self.load_methods(module, user_modules)
+            error_msg = u"{}: {}"
```

Comment:
It is there because I was trying to address unwanted `Import Error`. I later perceived this to be a #1619 bug.

If we fix `get_color` first (e.g., slicing off `get_color`, making our own `get_color`, or addressing errors differently), then this part, we can undo. For now, this is here temporarily to hide unrelated `Import Error, `